packages/app-types/index.d.ts

```diff
@@ -121,6 +121,84 @@ export interface VersionService {
   version: string;
 }
 
+export type FirebaseSignInProvider =
```

Comment:
These types are defined in `@firebase/util` in https://github.com/firebase/firebase-js-sdk/pull/4792. Can we just reuse them?

Reply:
Is it okay to introduce a new `@firebase/app-types` -> `@firebase/util` dependency for this?

Reply:
Yes, but why do they need to be in `@firebase/app-types`?

Reply:
I see what you mean now. I've modified `@firebase/firestore-types` to peer dep on `@firebase/util` and reuse the types.